app/controllers/genres_controller.rb

```diff
@@ -0,0 +1,25 @@
+class GenresController < ApplicationController
+  def index
+    @genres = Genre.all
+    @genre  = Genre.new
+  end
+
+  def show
+    @genre = Genre.find(params[:id])
+  end
+
+  def create
+    @genre = Genre.new(genre_params)
+    if @genre.save
+      redirect_to genres_path
+    else
+      redirect_to genres_path
```

Comment:
How would a user know whether their genre was created or not? Both the happy and sad paths do the same thing here.

Reply:
I see your reasoning for not scoping the create action to an admin, but we'd like you to implement that piece of the functionality none the less.